mozregression/metrics.yaml

```diff
@@ -27,3 +27,57 @@ usage:
     expires: never
     send_in_pings:
       - usage
+  build_type:
+    type: string
+    description: >
+      The build type being bisected (opt, shippable, etc.)
```

Comment:
Is the set of values small?  If so, maybe list them all here...?  Mostly just thinking about the data scientist that might look at the data someday...  (We need an enumerated type in Glean, but we don't yet have one).

Reply:
The list is constantly changing (e.g. `shippable` was only adding in the last year or so), so I don't think we can rely on enumeration. I could add a few more examples maybe to the list. Here's the current set mozregression supports:

```
(venv37) wlach@antwerp mozregression % mozregression --list-build-types
fennec:
  shippable
  opt
  pgo
  debug
fennec-2.3:
  shippable
  opt
  pgo
  debug
firefox:
  shippable
  opt
  pgo[linux32,linux64,win32,win64]
  debug
  asan[linux64]
  asan-debug[linux64]
gve:
  opt
  debug
jsshell:
  shippable
  opt
  pgo[linux32,linux64,win32,win64]
  debug
  asan[linux64]
  asan-debug[linux64]
thunderbird:
  opt
```

Reply:
Expanded the list a bit while still making it clear that there are other possibilities